### ISSUE TITLE
iOS support

### ISSUE THREAD

**Issue by cagnulein** [variar/klogg#468]
Any plans to port this to iOS?
I already did a similar job with https://github.com/cagnulein/qdomyos-zwift/ and I can give you a hand.

I think it will be handy for a lot of users!
What do you think? Do you see any major issues about this?

**Comment by variar** [comment]
@cagnulein you mean porting klogg to tablets? I don't think I'll do this, because I doubt there is a big demand for it. However, pull requests are always welcome.

**Comment by cagnulein** [comment]
@variar yes exactly, actually i'm grepping files with iSH app but it's a pain.
I see what I can do in the next weeks

**Comment by variar** [comment]
Klogg has some heavy external dependencies. Qt should work with iOS, but hyperscan and sentry should be disabled. There are options for it in cmake.

**Comment by luoqisheng** [comment]
I have write a tool to redirect iOS app stdout & stderr to macosx using usb so I can use klogg to read the log realtime
